app/controllers/api/v1/rooms_controller.rb

```diff
@@ -12,6 +12,9 @@ class RoomsController < ApiController
       before_action only: %i[create] do
         ensure_authorized('ManageUsers', user_id: room_params[:user_id])
       end
+      before_action only: %i[create] do
```

Comment:
This is fine for now,
But if we're gonna have a lot of this `AND` cases for the same actions as a pattern we may like to make a small refactoring to clean this up,